lib/features/email/presentation/controller/email_supervisor_controller.dart

```diff
@@ -61,13 +61,11 @@ class EmailSupervisorController extends GetxController {
   void createPageControllerAndJumpToEmailById(EmailId currentEmailId) {
     _currentEmailIndex = _currentListEmail.matchedIndex(currentEmailId);
     if (pageController != null && pageController?.hasClients == true) {
-      _jumpToPage(_currentEmailIndex);
+      pageController?.jumpToPage(_currentEmailIndex);
```

Comment:
for sure, pageController will not be null, so why do you use ? after pageController

Reply:
We can write in 2 ways it doesn't affect anything.

Reply:
:)